assets/src/stories-editor/blocks/amp-story-page-attachment/edit.js

```diff
@@ -81,42 +53,28 @@ class PageAttachmentEdit extends Component {
 			color: textColor.color,
 			backgroundColor: appliedBackgroundColor,
 		};
-		return {
-			style: attachmentStyle,
-			attachmentClass,
-		};
-	}
-
-	render() {
-		const {
-			attributes,
-			setAttributes,
-		} = this.props;
 
-		const {
-			openText,
-		} = attributes;
+		const newAttributes = { wrapperStyle: attachmentStyle };
+		newAttributes.attachmentClass = attachmentClass;
+		setAttributes( newAttributes );
+	}, [ colors, backgroundColor, customBackgroundColor, opacity, textColor, setAttributes ] );
```

Comment:
^ Here as well it looks like there are some unnecessary values tracked. E.g. `colors` (since it's called only once with `useSelect`, also, it doesn't really change either probably dynamically) and `setAttributes` which is also a static function.